src/main/java/io/jenkins/plugins/opentelemetry/semconv/JenkinsOtelSemanticAttributes.java

```diff
@@ -17,7 +17,8 @@
  * @see io.opentelemetry.semconv.trace.attributes.SemanticAttributes
  */
 public final class JenkinsOtelSemanticAttributes {
-	public static final AttributeKey<String>        CI_PIPELINE_TYPE = AttributeKey.stringKey("ci.pipeline.type");
+    public static final AttributeKey<String>        CI_PIPELINE_TYPE = AttributeKey.stringKey("ci.pipeline.type");
+    public static final AttributeKey<String>        CI_PIPELINE_TYPE_REF = AttributeKey.stringKey("ci.pipeline.type.ref");
```

Comment:
What does "_REF" stand for?

Reply:
Git reference, but I'm not 100% onboard with that value, do you think `ci.pipeline.multibranch.type` might be clear enough?

Reply:
Would it make sense to extend our `git.*` namespace?

We already have:
```
git.repository
git.branch
git.username
```
Maybe we could introduce `git.refspec`

I'm also wondering if the SCM attributes you want to report on the "root span" could just reuse the `git.*` attributes that we already collect on the `git` and `checkout` steps: if it's a multibranch pipeline, then I can report SCM attributes on the root span (see `workflowJob.getProperty(BranchJobProperty.class).getBranch().getSCM()` and `SCMHead.HeadByItem.findHead(run.getParent())`?

https://git-scm.com/book/en/v2/Git-Internals-The-Refspec

Reply:
Additional note: we probably want to add `git.url` that is today described via `git.repository` combined with `http.url` for git+https or `net.peer.name` for git+ssh.

Reply:
The initial idea was to facilitate the filtering transactions for multibranch pipelines in three different categories:
- branches
- change requests
- tags <--- no yet implemented

IIUC, the git.* namespace does not know about the above concepts which are specific for the multibranch pipelines or GitHub/GitLab. 

I see the value to extend the `git*` namespace, but for this particular case, I wanted a simpler category of values.

Do you think we could create an attribute with the type in the git namespace? Or might it cause some confusion?

Reply:
I would not introduce a jenkins specific concept in the `git.*` namespace. Let's go slowly and incrementally on this.